browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc

```diff
@@ -475,6 +480,43 @@ void BraveNewTabMessageHandler::HandleBrandedWallpaperLogoClicked(
   }
 }
 
+void BraveNewTabMessageHandler::HandleTriggerSponsoredRichMediaAdEvent(
+    const base::Value::List& args) {
+  AllowJavascript();
+
+  if (args.size() != 2) {
```

Comment:
this is an invariant, no? `CHECK`?

Reply:
Fixed. This was consistent with rest of class.